packages/e2e-test-utils-playwright/src/rest-api.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * External dependencies
+ */
+import fetch from 'node-fetch';
```

Comment:
More of a FYI, Playwright has now it's own Rest client: https://playwright.dev/docs/test-api-testing

Reply:
Yep yep, I've seen that in the docs, it's really cool! I had another [draft PR](https://github.com/WordPress/gutenberg/pull/38060) trying to refactor this file, it targets puppeteer though and I'll have to rewrite it into playwright and make use of `request` there.